bindings/swift/native/Cargo.toml

```
@@ -0,0 +1,22 @@
+[workspace]
+
+[package]
+name = "iota-wallet-swift-binding"
+version = "0.1.0"
+authors = ["Thoralf Müller <thoralf.mueller@iota.org", "Pasquale Ambrosini <pasquale.ambrosini@gmail.com"]
+edition = "2018"
+
+[lib]
+name = "wallet"
+crate-type = ["cdylib", "staticlib"]
+
+[dependencies]
+tokio = { version = "1.11", features = ["full"] }
+once_cell = "1.5.0"
+iota-wallet = { git = "https://github.com/iotaledger/wallet.rs", branch = "dev", default-features = false, features = ["stronghold"] }
+serde_json = "1.0"
+riker = "0.4"
+serde = "1.0"
+bee-common = "0.4.1"
```

Please upgrade to v0.5.0 as there is security problems.

```suggestion
bee-common = "0.5.0"
```